nodeclient/http_api_client.go

```diff
@@ -35,8 +35,40 @@ const (
 	// GET returns the node info.
 	RouteInfo = "/api/core/v3/info"
 
+	// RouteCongestion is the route for getting congestion details for the account.
+	// GET returns the congestion details for the account.
+	// MIMEApplicationJSON => json.
+	// MIMEVendorIOTASerializer => bytes.
+	RouteCongestion = "/api/core/v3/accounts/%s/congestion"
+
+	// RouteRewards is the route for getting the rewards for staking or delegation based on provieded output.
+	// GET returns the rewards for the output.
+	// MIMEApplicationJSON => json.
+	// MIMEVendorIOTASerializer => bytes.
+	RouteRewards = "/api/core/v3/rewards/%s"
+
+	// RouteStaking is the route for getting the information about current validators.
+	// GET returns the information about current validators.
+	// MIMEApplicationJSON => json.
+	// MIMEVendorIOTASerializer => bytes.
+	RouteStaking = "/api/core/v3/staking"
```

Comment:
I'm wondering if `/staking` is a descriptive endpoint, if the response is about the validators :thinking: